docs/SECURITY.md

```diff
@@ -0,0 +1,36 @@
+# Security Policy
+
+## Supported Versions
+
+We release patches for security vulnerabilities for latest software versions.
+
+## Reporting a Vulnerability
+
+The security team and community take all security vulnerabilities seriously. 
+Thank you for improving the security of our open source software. 
+We appreciate your efforts and responsible disclosure and will make every effort to acknowledge your contributions.
+
+Report security vulnerabilities by emailing the security team at:
+
+```
```

Comment:
replcase with the group name

Reply:
We will consider that in future, add a new security template, add the new info here.
Anyway, the SECURITY.md file has a standard naming and is recommended.